src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/GetApprenticeshipsRequestMapper.cs

```diff
@@ -40,12 +40,20 @@ public async Task<ManageApprenticesViewModel> Map(GetApprenticeshipsRequest sour
 
             var filters = new GetApprenticeshipsFilterValuesResponse();
```

Comment:
It feels odd to be instantiating an api response type here.

Reply:
Moved this further down